experimental/packages/opentelemetry-api-metrics/src/types/Metric.ts

```diff
@@ -125,4 +125,4 @@ export type ObservableCounter = ObservableBase;
 /**
  * key-value pairs passed by the user.
  */
-export type Attributes = { [key: string]: string };
+export type Attributes = { [key: string]: string } | null;
```

Comment:
why does the attributes can be null ? I believe we are moving to use a common definitions for both trace & metrics and since traces can't be null i don't think this will works

Reply:
I thought null is used as a default value but it appears empty object is used. I will update this.